include/seccomp.h.in

```diff
@@ -65,6 +65,7 @@ enum scmp_filter_attr {
 	SCMP_FLTATR_CTL_TSYNC = 4,	/**< sync threads on filter load */
 	SCMP_FLTATR_API_TSKIP = 5,	/**< allow rules with a -1 syscall */
 	SCMP_FLTATR_CTL_LOG = 6,	/**< log not-allowed actions */
+	SCMP_FLTATR_SPEC_ALLOW = 7, /**< disable SSB mitigation */
```

Comment:
You'll also need to update the `doc/man/man3/seccomp_attr_set.3` manpage for the new filter attribute.

Reply:
Also, while you don't need to go into great detail in the manpage, please spell out SSB.  Several years from now, I can see some users forgetting about all the speculative store problems and scratching their head about the SSB acronym :)

Reply:
Sorry, one more thing: you also need to update `src/python/libseccomp.pxd`.